### DISCUSSION TITLE
How do I change the background color of a ScrollArea?

### DISCUSSION THREAD

**Discussion by shirakaba** [nodegui/svelte-nodegui#47] [Q&A]
Copying this Spectrum post from @johanalkstal:

> It's always gray and setting background-color on the ScrollArea or its contentWidget doesn't seem to do anything?

**Comment by shirakaba** [comment]
I've just realised this is more of a NodeGUI question than a Svelte NodeGUI one. We can discuss it here until NodeGUI sets up GitHub Discussions, I suppose.

**Reply by johanalkstal** [reply]
Thanks for picking up my question!

**Reply by shirakaba** [reply]
I don't see any special property for setting background colour (i.e. without CSS) in the API docs: https://docs.nodegui.org/docs/api/generated/classes/qscrollarea#widget

No mention of anything similar in the API docs for the underlying QScrollArea, either: https://doc.qt.io/qt-5/qscrollarea.html

**Reply by johanalkstal** [reply]
Thing is I've seen applications through tutorials online use it that don't have its gray background, yet I get this ugly gray color. 😢 

Been trying to put the background-color property through inline style on different components without any luck.
